### PR TITLE
helm chart: allow persistentVolumeClaim in psp or pod never launches

### PR DESCRIPTION
**Pull request by crookedstorm**
Simple fix, but if you have podsecuritypolicy in your cluster, this chart doesn't work without this change.

### CODE REVIEW

**Comment by crookedstorm** [comment]
Sorry, figuring out the system and jumped the gun on the assigning. I'll wait for the approval and such and leave the bot alone for now. Thanks! 😁

**Comment by yonatankahana** [comment]
hi, 
when changing the chart templates it is require to also bump the chart version in `Chart.yaml`. merging this PR will trigger release pipeline that will fail unless it is done.

/assign

**Comment by crookedstorm** [comment]
> hi,
> when changing the chart templates it is require to also bump the chart version in `Chart.yaml`. merging this PR will trigger release pipeline that will fail unless it is done.
> 
> /assign

Done, I think.

**Review by yonatankahana — APPROVED** [review]

**Comment by crookedstorm** [comment]
So this needs an 'lgtm' command from a maintainer as well I think. Anything else needed from me for that? Would it be helpful for me to go find people or should I simply find patience? 😄

**Comment by yonatankahana** [comment]
> So this needs an 'lgtm' command from a maintainer as well I think. Anything else needed from me for that? Would it be helpful for me to go find people or should I simply find patience? smile

there is another pipeline that waiting for approval so I tried to re-approve but for some reason it is still waiting for something. its not that important but i preferred to wait.  

anyway,
/lgtm

**Comment by yonatankahana** [comment]
@crookedstorm thank you for your contribution :)

**Pull request merged**
